assets/scss/modules/_forms.scss

```diff
@@ -266,55 +266,116 @@ input[type="tel"] {
   }
 }
 
-.wrapper-errors {
-  display: none;
+/* Mixin for dp-message */
+/// @access private
+@mixin dp-message($dp-font-color, $dp-font-size) {
+    a, p, button, span{
+        font-size: $dp-font-size;
+        color:$dp-font-color;
+        text-align: left;
+        margin: 0;
+    }
 }
 
-.form-message {
-  margin-bottom: $dp-spaces--lv2;
+/// Form messages and validations
+/// @content [Form messages and validations]
+/// @access public
+/// @deprecated .error and .wrapper-errors will be removed in a next version of the library
+/// @author gbaamonde@makingsense.com
+
+
+/* Message modules */
+
+.dp-message, .error {
+  display: none;
 }
 
-.error {
+/* Class for validation */
+
+.dp-validate, .error {
   .checkmark,
   input,
   select,
   textarea {
     border: 1px solid $dp-color-red;
   }
 
-  .wrapper-errors {
+/* common properties */
+  .dp-message-box {
+    padding-left: $dp-sizing--lvl10;
+    min-height: 45px;
+    display: flex;
+    margin: $dp-spaces--lv2 $dp-spaces--lv0;
+    flex-direction: column;
+    justify-content: center;
+  }
+/* Messages and confirmations the forms */
+  .dp-message, .wrapper-errors {
     display: inline-block;
     margin: $dp-spaces--lv0;
-    color: $dp-color-red;
-    font-size: $dp-sizing--lvl2;
     font-family: $dp-font-family-proximanova;
+    color: $dp-color-red;
+    font-size: 13px;
 
-    a {
-      color: $dp-color-red;
-      text-decoration: underline;
+      a {
+        text-decoration: underline;
+      }
+      button {
+        text-decoration: underline;
+      }
+      p {
+        margin: $dp-spaces--lv0;
+      }
+    /* Validations error message */
+    &.dp-error-form {
+      @include dp-message($dp-color-red, $dp-sizing--lvl2);
     }
+  }
+}
 
-    button {
-      color: $dp-color-red;
-      text-decoration: underline;
-    }
 
-    p {
-      margin: $dp-spaces--lv0;
-      font-size: $dp-sizing--lvl2;
-      line-height: $dp-spaces--lv3;
-      font-family: $dp-font-family-proximanova;
-    }
-  }
 
+/* Unexpected message box */
+.form-message {
+  margin-bottom: $dp-spaces--lv2;
+  display: flex;
+  min-height: 45px;
+  flex-direction: column;
+  justify-content: center;
+
+/* Validations Unexpected message */
+&.dp-error {
```

Comment:
Both `.dp-error` and `.dp-ok-message` are classes that can be simplified since both share most properties except `background-image`. How about something like this:
```css
&.dp-error,
&.dp-ok-message {
  @extend .dp-message-box;
  @include dp-message($dp-color-red, $dp-sizing--lvl3); //mixins should be included before properties since in that way any property that comes from the mixin can be overridden
  
  padding-left: $dp-sizing--lvl10;
  background-repeat: no-repeat;
  background-position: 0;
}

&.dp-error {
  background-image: url("../img/error-message.svg");
}

&.dp-ok-message {
  background-image: url("../img/ok-message.svg");
}
```

Reply:
you have the whole reason, thanks @damianmuti !